releases/datadog-agent/datadog-agent-1.yml

```diff
@@ -0,0 +1,19 @@
+name: datadog-agent
+version: "1"
```

Comment:
hmm shouldn't this be a version that complies with our versioning scheme?
By the way what's the final versioning scheme we decided to keep? (I think the REAME section on that is not very clear, looks like the versioning scheme and its explanation contradict each other)